src/analysis/retail/monk/mistweaver/modules/spells/Upwelling.tsx

```diff
@@ -86,12 +86,8 @@ class Upwelling extends Analyzer {
   }
 
   get lostEfHealing() {
-    if (this.averageStacks < 12) {
-      // number of bolts is 1:1 with normal ef if less than 12 stacks
-      return 0;
-    }
-    const wastedBolts = this.averageStacks - 12;
-    return this.baseEfHealing * (1 - wastedBolts / 6);
+    const lostBolts = this.averageStacks / 2; // you lose .5 bolts a second
+    return this.baseEfHealing * (lostBolts / BASE_BOLTS);
```

Comment:
does this make the module count bolts that aren't added by upwelling for the lost healing? not sure if thats relevant

Reply:
Yes it does, because upwelling stacks equate to lost casts